### DISCUSSION TITLE
Which EEPROM/FRAM chip

### DISCUSSION THREAD

**Discussion by JJTechman68** [phil-barrett/PicoCNC#22] [General]
I purchased the new 1.53eth board and an FM24CL16B-GTR chip to solder onto it, which was mentioned in the Teensy discussion but apparently is not the chip for this board, as the #1 positions do not line up. Which chip do I need?

**Comment by phil-barrett** [comment]
Hmmm,  that is the right part number. The package should be a SOIC-8 chip.  There is a DFN version, did you get that?

**Comment by JJTechman68** [comment]
I received this chip: https://www.digikey.ca/en/products/detail/infineon-technologies/FM24CL16B-GTR/3788934

**Reply by phil-barrett** [reply]
That's the correct chip.  Can you show me a photo of what you are talking about (pin 1 not lining up)?

**Comment by JJTechman68** [comment]
So if this is the board, https://github.com/phil-barrett/PicoCNC/blob/main/PicoCNC%20V1.53eth%20cropped.jpg?raw=true , then pin 1 would be beside the U for U18 or at the top pad above U18? I have been using the small indent beside the U as #1, and on the chip #1 is on the opposite side 

![428_8-SOIC](https://github.com/phil-barrett/PicoCNC/assets/76827669/6bbc3e54-993d-46ab-b45b-08a106963a84)
![PicoCNC V1 53eth cropped](https://github.com/phil-barrett/PicoCNC/assets/76827669/0ce37d7c-5e90-47ea-affa-15b908501937)

**Reply by phil-barrett** [reply]
Align the chip so the pin 1 mark coincides with the little circle silk screened inside the chip outline.  See the marked up images. You will need to build a new grblHAL with EEProm enabled. It's on the Plug-Ins page of the builder.  Select 24x16 and check the FRAM box.

![image](https://github.com/phil-barrett/PicoCNC/assets/16570958/688a364f-a26d-4897-b250-76299763f317)
![image](https://github.com/phil-barrett/PicoCNC/assets/16570958/26acd32f-00fe-4bec-b02a-4eac31043933)

**Comment by JJTechman68** [comment]
Thanks Phil. You should add this pic to your manual along with the EEPROM/FRAM number. Hopefully I'm not the only challenged one here!

**Reply by phil-barrett** [reply]
The little indent you took as the pin 1 mark is actually a via that connects the top layer to the bottom. I can see how that is confusing.  Thanks for the input on the manual.

Phil
